var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -24,6 +24,12 @@ class PyProtobuf(PythonPackage):
     version("4.21.9", sha256="61f21493d96d2a77f9ca84fefa105872550ab5ef71d21c458eb80edcf4885a99")
     version("4.21.7", sha256="71d9dba03ed3432c878a801e2ea51e034b0ea01cf3a4344fb60166cb5f6c8757")
     version("4.21.5", sha256="eb1106e87e095628e96884a877a51cdb90087106ee693925ec0a300468a9be3a")
+    version(
+        "3.20.3",
+        sha256="2e3427429c9cffebf259491be0af70189607f365c2f41c7c3764af6f337105f2",
+        preferred=True,
+    )
```

Comment:
No more preferred version.
```suggestion
    version("3.20.3", sha256="2e3427429c9cffebf259491be0af70189607f365c2f41c7c3764af6f337105f2")
```